src/_tutorials/language/futures.md

```diff
@@ -367,17 +381,21 @@ written in a couple different ways.
 
   <?code-excerpt "misc/lib/tutorial/daily_news.dart (main-future-api-pass-print)"?>
   {% prettify dart %}
-  printDailyNewsDigest() =>
+  Future<void> printDailyNewsDigest() =>
       gatherNewsReports().then(print);
   {% endprettify %}
 
+If you use `.then()` on a `Future<void>`, you still need to specify a parameter
```

Comment:
Let's rewrite this slightly. I don't think we need to sell them on the feature, but it'd be good to _show_ them how to use it. Maybe something like this:

----------------

If you use `then()` on a `Future<void>` object, you still need to specify a parameter to `then()`'s callback:

```
final future = printDailyNewsDigest();
return future.then((_)) { // Unused parameter (named `_`, by convention)
  // Code that doesn't use the `_` parameter...
});
```

----------------

It'd be best if we could add this example to the examples/ directory, so we can check the syntax.